### PR TITLE
HashId Fix and Babel

### PR DESCRIPTION
**Pull request by mpalmr**
- Added build step for Babel to support front end projects using this project as a dependencies
- Used default export of `hashids` dependency

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,12 +2,17 @@
   "name": "@compassdigital/id",
   "version": "2.2.0",
```
Comment (jpdemiranda) [inline package.json]: Can you update the version please

File: index.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const Hashids = require('hashids');
+const Hashids = require('hashids').default;
```
Comment (rcjpisani) [inline index.js]: Is using the `default` property on the hashids package going to break anything? I can't seem to see a reference to that property in their docs.

Comment (mpalmr) [reply index.js]: They're using ESM's `export default` now. When this translates to CJS, you get an object that looks like `{ default: HashIds }`

File: package.json (hunk 1 above)
Comment (mpalmr) [reply package.json]: Done

**Pull request merged**